assets/php/class-wgpb-block-library.php

```diff
@@ -335,6 +347,17 @@ public static function render_product_on_sale( $attributes, $content ) {
 		$block = new WGPB_Block_Product_On_Sale( $attributes, $content );
 		return $block->render();
 	}
+
+	/**
+	 * Add body classes.
+	 *
+	 * @param array $classes Array of CSS classnames.
+	 * @return array Modified array of CSS classnames.
+	 */
+	public static function add_theme_body_class( $classes = array() ) {
+		$classes[] = 'theme-' . get_stylesheet();
```

Comment:
Maybe this should be `get_template()` so that storefront child themes get the storefront styles too?